test/common/init/target_impl_test.cc

```diff
@@ -63,20 +63,26 @@ TEST(InitTargetImplTest, ReadyWhenWatcherUnavailable) {
 }
 
 TEST(SharedTargetImplTest, InitializeTwiceBeforeReady) {
+  InSequence s;
   ManagerImpl m1("m1");
   ManagerImpl m2("m2");
   ExpectableWatcherImpl w1;
   ExpectableWatcherImpl w2;
   ExpectableSharedTargetImpl t;
+  //t.expectInitialize();
   m1.add(t);
-  EXPECT_EQ(0, t.count_);
+  // @mergeconflit: I want to verify Initalize invoked after m2.add(t). 
+  // But if I swap the below lines the test is also passing.
+  // 
+  // t.expectInitialize();
+  // m2.add(t);
   m2.add(t);
-  EXPECT_EQ(0, t.count_);
+  t.expectInitialize();
```

Comment:
The test would pass if I uncomment 77,78 and comment 79,80.
However, my expectation is that sequence should fail.

I think the InSequence make sure ONLY the EXPECT_CALLs are executed in the defined order.
Since there is no other EXPECT_CALL among these 2 lines the InSequence doesn't help. 
I don't want to introduce another EXPECT_CALL in add()